xml/en/docs/njs/install.xml

```diff
@@ -69,6 +69,21 @@ The modules can also be built as
 </example>
 </para>
 
+<para>
+To build with <link doc="engine.xml">QuickJS</link> engine
+support clone the QuickJS repository:
+<example>
+git clone https://github.com/bellard/quickjs
+</example>
+and provide include and library path using <literal>--with-cc-opt=</literal> and
```

Comment:
Don't you have to build quickjs first, and then provide flags to nginx configure?